Spring Boot testcase error

## Bug Report

Ref https://travis-ci.org/github/apache/shardingsphere-elasticjob/builds/739336174

### Actual behavior
```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.apache.shardingsphere.elasticjob.lite.spring.boot.job.ElasticJobSpringBootTest
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.77 sec <<< FAILURE! - in org.apache.shardingsphere.elasticjob.lite.spring.boot.job.ElasticJobSpringBootTest
initializationError(org.apache.shardingsphere.elasticjob.lite.spring.boot.job.ElasticJobSpringBootTest)  Time elapsed: 0.01 sec  <<< ERROR!
java.lang.IllegalStateException: Could not load TestContextBootstrapper [null]. Specify @BootstrapWith's 'value' attribute or make the default bootstrapper class available.
Caused by: java.lang.NoClassDefFoundError: org/springframework/util/ConcurrentLruCache
Caused by: java.lang.ClassNotFoundException: org.springframework.util.ConcurrentLruCache
Results :
Tests in error: 
  ElasticJobSpringBootTest.initializationError » IllegalState Could not load Tes...
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
```

--- a/elasticjob-lite/elasticjob-lite-spring/pom.xml
+++ b/elasticjob-lite/elasticjob-lite-spring/pom.xml
@@ -34,7 +34,7 @@
 
     <properties>
         <springboot.version>2.3.1.RELEASE</springboot.version>
-        <springframework.version>[3.1.0.RELEASE,)</springframework.version>
+        <springframework.version>[3.1.0.RELEASE,5.2.7.RELEASE]</springframework.version>
     </properties>
 
     <dependencyManagement>

